semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/BindingMethodOverridesPass.scala

```diff
@@ -0,0 +1,39 @@
+package io.shiftleft.semanticcpg.passes
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.{NodeKeys, nodes}
+import io.shiftleft.passes.{CpgPass, DiffGraph}
+import io.shiftleft.semanticcpg.language._
+import org.apache.logging.log4j.LogManager
+
+class BindingMethodOverridesPass(cpg: Cpg) extends CpgPass(cpg) {
+  override def run(): Iterator[DiffGraph] = {
+    cpg.method.map(processMethod).toList().iterator
+  }
+
+  def processMethod(m: nodes.Method): DiffGraph = {
```

Comment:
No single letter abbreviations please. `method` seems to be a more appropriate name.

Reply:
Done